capi/VideoDecoderCapi.h

```diff
@@ -20,8 +20,8 @@
 #ifndef __ENABLE_CAPI__
 #define __ENABLE_CAPI__ 1
 #endif
-#include "interface/VideoDecoderDefs.h"
-#include "interface/VideoCommonDefs.h"
+#include "VideoDecoderDefs.h"
+#include "VideoCommonDefs.h"
```

Comment:
Yeah, other things I should mention. We will create a header file named LibyamiC.h, it will include all capi related file. You only need include this file in your project in future.